sdk/src/schema/webhook.ts

```diff
@@ -0,0 +1,17 @@
+import type { MergeCoreCollection } from '../index.js';
+
+export type DirectusWebhook<Schema extends object> = MergeCoreCollection<
+	Schema,
+	'directus_webhooks',
+	{
+		id: number;
+		name: string;
+		method: string;
+		url: string;
+		status: string;
+		data: boolean;
+		actions: string;
+		collections: string;
```

Comment:
Fields of type `csv` always seem to return an array not a string while accepting both, an array and string for requests. At least confirmed in this case for `createWebhook`:
```js
// requests
actions: 'create'
actions: ['create']
actions: 'create,delete'
actions: ['create', 'delete']

// responses
actions: [ 'create' ]
actions: [ 'create' ]
actions: [ 'create', 'delete' ]
actions: [ 'create', 'delete' ]
```

This means the response type is clearly wrong. However, I'm not sure how we can best deal with something like this, since request / response types are different 🤔